src/hypofuzz/hypofuzz.py

```diff
@@ -283,10 +343,43 @@ def valid_targets(self) -> list[FuzzTarget]:
         # the targets we actually want to run/fuzz
         return [t for t in self.targets if not t.has_found_failure]
 
+    def _maybe_add_targets(self) -> None:
+        # consider whether it's worthwhile to add more targets
+        active_nodeids = {target.nodeid for target in self.targets}
+        candidates = [nodeid for nodeid in self.nodeids if nodeid not in active_nodeids]
+        # TODO actually defer starting up targets here, based on worker lifetime
+        # and startup cost estimators here
+        for nodeid in candidates:
+            self.add_target(nodeid)
```

Comment:
I think I'd like to move this responsibility into the hub process; it can slowly 'feed in' extra targets to amortize startup cost, etc.  That way only one process needs to read from the database to get good performance, and we get a pretty nice separation of concerns too.